lib/std/os/linux.zig

```diff
@@ -6816,6 +6816,139 @@ pub const ifreq = extern struct {
     },
 };
 
+pub const PACKET = struct {
+    pub const HOST = 0;
+    pub const BROADCAST = 1;
+    pub const MULTICAST = 2;
+    pub const OTHERHOST = 3;
+    pub const OUTGOING = 4;
+    pub const LOOPBACK = 5;
+    pub const USER = 6;
+    pub const KERNEL = 7;
+    pub const FASTROUTE = 8;
```

Comment:
This is incorrect; it's 6 in the UAPI headers.

Reply:
Eeep, so it is. Kinda weird collision there with USER?

I'm fine with removing it as well though; I'll do so.